skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -178,7 +177,7 @@ def _nl_means_denoising_2d(image, int s=7, int d=13, float h=0.1):
     cdef int row_start, row_end, col_start, col_end
     cdef int row_start_i, row_end_i, col_start_j, col_end_j
     cdef IMGDTYPE [::1] new_values = np.zeros(n_ch).astype(np.float32)
-    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(util.pad(image,
+    cdef IMGDTYPE [:, :, ::1] padded = np.ascontiguousarray(np.pad(image,
```

Comment:
I'm a big +1 with Josh here. When I talk to people, they mix so many libraries (opencv, pillow, scikit-image) to get the easiest way to solve their problem. Often, it's just because they don't have a deep understanding of the python stack. 

In a general manner, I think we need to implement very simple functions (I added `invert` recently) that may (or may not) look stupid for us, but that will improve the user experience significantly.